src/components/header/header.js

```diff
@@ -7,26 +7,32 @@ class Header extends HTMLElement {
     this.links = [
       {
         name: 'Home',
-        href: 'home.html',
-        active: this.checkPathname('index.html'),
+        href: 'home/',
+        active: this.checkPathname('home'),
       },
       {
         name: 'Films',
-        href: 'films.html',
-        active: this.checkPathname('films.html'),
+        href: 'pages/films/',
+        active: this.checkPathname('films'),
       },
       {
         name: 'Collections',
-        href: 'collections.html',
-        active: this.checkPathname('collections.html'),
+        href: 'pages/collection-details/',
+        active: this.checkPathname('collection-details'),
       },
     ];
   }
 
   // eslint-disable-next-line class-methods-use-this
```

Comment:
jesteśmy w stanie odwołać się do window bez blokowania eslinta?

Reply:
z tego co pisaliśmy na discord to prawdopodobnie nie